### PR TITLE
Fix typos in basics/ranges

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: public/content/en/basics/ranges.md [hunk 1]
```diff
@@ -4,28 +4,27 @@ If a `foreach` is encountered by the compiler
...
 
-The functions that are in the `std.range` and `std.algorithm` modules also provide
+The functions that are in the `std.range` and `std.algorithm` modules provide
 building blocks that make use of this interface. Ranges allow
 to compose complex algorithms behind an object that
-can be iterated with ease. And ranges allows to create **lazy**
-objects that actually just perform a calculation when this
-is really needed in an iteration e.g. when a range's
-element is used.
+can be iterated with ease. Furthermore allow ranges to create **lazy**
```
Comment (stonemaster) [inline public/content/en/basics/ranges.md]: "Furthermore ranges allow" ? I am unsure. "Furthermore allow ranges" sounds a little bit German :-)

**Comment by wilzbach** [comment]
(rebased)

File: public/content/en/basics/ranges.md [hunk 2]
```diff
@@ -4,28 +4,27 @@ If a `foreach` is encountered by the compiler
...
-.. it's rewritten to something equivalent to the following internally:
+it's internally rewritten similar to the following:
 
     for (; !range.empty; range.popFront()) {
         auto element = range.front;
         ...
 
 Any object which fulfills the above interface is called a **range**
-and is thus something that can be iterated over:
+and is thus an object that can be iterated over:
```
Comment (JackStouffer) [inline public/content/en/basics/ranges.md]: "a type" would be more accurate

File: public/content/en/basics/ranges.md [hunk 3]
```diff
@@ -4,28 +4,27 @@ If a `foreach` is encountered by the compiler
...
 
     struct Range {
         @property empty() const;
         void popFront();
         T front();
     }
 
-The functions that are in the `std.range` and `std.algorithm` modules also provide
+The functions that are in the `std.range` and `std.algorithm` modules provide
```
Comment (JackStouffer) [inline public/content/en/basics/ranges.md]: You can just say "The functions in `std.range` and `std.algorithm` provide"

**Comment by wilzbach** [comment]
thanks @JackStouffer - rebased ;-)

**Pull request merged**